### DISCUSSION TITLE
Can I Join The Team?

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#16417] [Q&A]
Github: https://github.com/OmegaBeam69
Telegram: https://t.me/OmegaBeam
Recent Successful Builds: 
https://cirrus-ci.com/task/5188616847097856
https://cirrus-ci.com/task/4954478181351424
https://cirrus-ci.com/build/4925241030344704
https://cirrus-ci.com/build/6308519670972416
DT: https://github.com/OmegaBeam69/WF_device_xiaomi_raphael/tree/spark-13
KT: https://github.com/OmegaBeam69/bool-x_xiaomi_raphael
VT: 
https://github.com/OmegaBeam69/vendor_xiaomi_raphael

**Comment by TenMoons** [comment]
你好，邮件我已收到。

**Comment by Wintere107** [comment]
收到邮件

**Comment by OmegaBeam69** [comment]
@Apon77 bhai can you help please? Thank you.
